package.json

```diff
@@ -15,7 +15,7 @@
     "loader/"
   ],
   "scripts": {
-    "build": "stencil build --docs",
+    "build": "stencil build --docs-json component-docs.json",
```

Comment:
Building this json file will make it easy to include component documentation in our main Storybook instance.